arkouda/dataframe.py

```diff
@@ -1149,6 +1570,29 @@ def reset_index(self, size: bool = False, inplace: bool = False) -> Union[None,
     def info(self):
         """
         Returns a summary string of this dataframe.
+
+        Returns
+        -------
+        str
+            A summary string of this dataframe.
+
+        Examples
+        --------
+
+        >>> df = ak.DataFrame({'col1': [1, 2], 'col2': ["a", "b"]})
+        >>> df
+
+        +----+--------+--------+
+        |    |   col1 | col2   |
+        +====+========+========+
+        |  0 |      1 | a      |
+        +----+--------+--------+
+        |  1 |      2 | b      |
+        +----+--------+--------+
```

Comment:
The formatting of col2 is different from everything else in here

Reply:
I used the to_markdown function from pandas dataframes.  It shouldn't make a difference when rendered in sphinx.  Sphinx makes an actual table graphic for these.

Reply:
![dataframe_docs_screenshot](https://github.com/Bears-R-Us/arkouda/assets/11051061/e29454d0-51ff-49fe-ab72-16f68e933ea1)

Reply:
Included a screenshot to show they render the same in sphinx.